httpserver/src/main/kotlin/tbdex/sdk/httpserver/handlers/CreateExchange.kt

```diff
@@ -27,23 +29,37 @@ import tbdex.sdk.protocol.models.Rfq
  * @param callback An optional callback function to be invoked after processing the RFQ.
  */
 @Suppress("TooGenericExceptionCaught", "SwallowedException")
-suspend fun submitRfq(
+suspend fun createExchange(
   call: ApplicationCall,
   offeringsApi: OfferingsApi,
   exchangesApi: ExchangesApi,
   callback: SubmitCallback?
 ) {
   val message: Rfq?
-
+  val replyTo: String?
   try {
-    message = Message.parse(call.receiveText()) as Rfq
+    val requestBody = call.receiveText()
+
+    val jsonNode = Json.jsonMapper.readTree(requestBody)
+    val rfqJsonString = jsonNode["rfq"].toString()
+
+    message = Message.parse(rfqJsonString) as Rfq
+    replyTo = jsonNode["replyTo"]?.takeIf { !it.isNull }?.asText()
```

Comment:
```suggestion
    replyTo = jsonNode["replyTo"]?.asText()
```

Reply:
actually, reverting this - i think i tried it the way you suggested but it keeps populating replyTo as `"null"` as in stringified `null`